ext/gvl_tracing_native_extension/gvl_tracing.c

```diff
@@ -169,7 +175,7 @@ static void render_event(const char *event_name) {
     render_thread_metadata();
   }
 
-  unsigned int thread_id = current_thread_serial;
+  uint64_t thread_id = native_thread_id();
```

Comment:
Rather that retrieving it on every call, I suggest storing the `native_thread_id` in a `_Thread_local` like we're doing for the serial, and then initializing it only once (e.g. similar to what we're doing with the serial).

This way we avoid that potential overhead for each event emission :)